puppet/modules/kubernetes/templates/kubelet.service.erb

```diff
@@ -65,10 +73,6 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/kubelet \
   --authorization-mode=Webhook \
 <% end -%>
 <% # Kubernetes older than 1.7 has issues with kernel 4.9+ -%>
-<% if not @kernelversion.nil? and scope.function_versioncmp([scope['kubernetes::version'], '1.6.0']) >= 0 and scope.function_versioncmp([scope['kubernetes::version'], '1.7.0']) < 0 and scope.function_versioncmp([@kernelversion, '4.9']) >= 0 -%>
-  --cgroups-per-qos=false \
-  --enforce-node-allocatable= \
-<% end -%>
```

Comment:
Please do not make such behavioural changes without noting it in the release notes. Also it does not hurt keeping it in, does it?

Reply:
Sorry you still have it in, why all this order changes?